### PR TITLE
Allow user to mark certain tasks as important  

### PR DESCRIPTION
**Pull request by SukiTsang**

### CODE REVIEW

File: src/main/java/seedu/task/logic/commands/UpdateCommand.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public CommandResult execute() {
         
         Task newTask = new Task(
                 updatedTaskName,
-                newTaskTags
+                newTaskTags,false
```
Comment (Skaty) [inline src/main/java/seedu/task/logic/commands/UpdateCommand.java]: Why `false`?

**Review by Skaty — CHANGES_REQUESTED** [review]
I'll fix your indentation issues.

File: src/main/java/seedu/task/logic/commands/UpdateCommand.java (hunk 1 above)
Comment (SukiTsang) [inline src/main/java/seedu/task/logic/commands/UpdateCommand.java]: since a new task will not set to be important

**Comment by SukiTsang** [comment]
@Skaty I also want to learn about how to do the indentation since I'm using the format in eclipse only. Maybe can you point out my indentation issue and how can I improve? So I can improve next time.

File: src/main/java/seedu/task/logic/commands/UpdateCommand.java (hunk 1 above)
Comment (SukiTsang) [reply src/main/java/seedu/task/logic/commands/UpdateCommand.java]: Fixed

**Comment by Skaty** [comment]
@SukiTsang you need to ensure that it is not tabs, but 4 spaces. Did you set up Eclipse to use 4 spaces instead of tab?

**Comment by SukiTsang** [comment]
I set that before. There should have something wrong. Fixed already. Thanks!

**Review by Skaty — APPROVED** [review]

**Pull request merged**